src/index.jsx

```diff
@@ -22,11 +23,12 @@ ReactDOM.render(
     <HashRouter>
       <div>
         <Switch>
+          <Route path={['/', '/login']} exact component={Login} />
```

Comment:
Currently PropTypes throws an error when you send an array of strings for the path, consider using a regex expression for the path instead.

Reply:
You should address this in the other PR, this code shouldn't be here in the first place.